### PR TITLE
Update data table styles 

### PR DESCRIPTION
**Pull request by daniel-tabarcea**
- updated row shadows on data tables
- updatd fund reservations table

### CODE REVIEW

**Comment by daniel-tabarcea** [comment]
connects unicef/etools-issues#855

File: pmp/app-elements/interventions/fr-numbers/fund-reservations-display.html [hunk 1]
```diff
@@ -1,56 +1,88 @@
...
-        <div class="col col-2 right-align">
-          <etools-info-tooltip icon="report"
-                            hide-tooltip$="[[!frsConsistencyWarningIsActive(_frsTotalAmountWarning)]]"
-                            important-warning
-                            icon-first right-aligned>
-            <span slot="field">[[displayCurrencyAmount(frsDetails.total_intervention_amt, '0.00')]]</span>
-            <span slot="message">[[_frsTotalAmountWarning]]</span>
-          </etools-info-tooltip>
+
+      <template is="dom-if" if="[[!frsDetails.frs.length]]">
```
Comment (adi130987) [inline pmp/app-elements/interventions/fr-numbers/fund-reservations-display.html]: Put this warning before `etools-data-table-header`. Right now is in the middle of the template :)

File: pmp/app-elements/interventions/fr-numbers/fund-reservations-display.html [hunk 2]
```diff
@@ -1,56 +1,88 @@
...
+          <div slot="row-data">
+              <span class="col-data col-3">[[fr.fr_number]]</span>
+              <span class="col-data col-3 right-align">[[prettyDate(fr.start_date)]]</span>
+              <span class="col-data col-2 right-align">[[displayCurrencyAmount(fr.intervention_amt, '0.00')]] </span>
+              <span class="col-data col-2 right-align">[[displayCurrencyAmount(fr.actual_amt, '0.00')]]</span>
+              <span class="col-data col-2 right-align">[[displayCurrencyAmount(fr.outstanding_amt, '0.00')]]</span>
+          </div>
+        </etools-data-table-row>
       </template>
-      <div class="row-h row-second-bg">
```
Comment (adi130987) [inline pmp/app-elements/interventions/fr-numbers/fund-reservations-display.html]: keep totals row background color, it was in desing, we needed back.

File: pmp/app-elements/interventions/fr-numbers/fund-reservations-display.html [hunk 3]
```diff
@@ -1,56 +1,88 @@
...
+
       <template is="dom-repeat" items="[[frsDetails.frs]]" as="fr">
-        <div class="row-h">
-          <div class="col col-3">[[fr.fr_number]]</div>
-          <div class="col col-3 right-align">[[prettyDate(fr.start_date)]]</div>
-          <div class="col col-2 right-align">[[displayCurrencyAmount(fr.intervention_amt, '0.00')]] </div>
-          <div class="col col-2 right-align">[[displayCurrencyAmount(fr.actual_amt, '0.00')]]</div>
-          <div class="col col-2 right-align">[[displayCurrencyAmount(fr.outstanding_amt, '0.00')]]</div>
-        </div>
+        <etools-data-table-row no-collapse secondary-bg-on-hover>
```
Comment (adi130987) [inline pmp/app-elements/interventions/fr-numbers/fund-reservations-display.html]: I do not think we need `secondary-bg-on-hover` here, you do not interact with this data.

File: pmp/app-elements/interventions/reports/reporting-requirements.html [hunk 4]
```diff
@@ -34,11 +34,15 @@
       }
 
       .actions-button-container paper-icon-button {
-        opacity: 0;
+        /* height: 32px;
```
Comment (adi130987) [inline pmp/app-elements/interventions/reports/reporting-requirements.html]: remove this unused css

File: pmp/app-elements/interventions/fr-numbers/fund-reservations-display.html (hunk 2 above)
Comment (daniel-tabarcea) [reply pmp/app-elements/interventions/fr-numbers/fund-reservations-display.html]: I couldn't use the same class, it doesn't work with the data tables. But I've added the required css through a mixin.

**Pull request merged**